flink/v1.13/flink/src/test/java/org/apache/iceberg/flink/TestHelpers.java

```diff
@@ -67,6 +71,16 @@
   private TestHelpers() {
   }
 
+  public static <T> T roundTripKryoSerialize(Class<T> clazz, T table) throws IOException {
```

Comment:
I am wondering if this should stay in `TestTableSerialization` class only.

Reply:
I put this round trip kryo serialize method in this flink TestHelpers, because I think other object that needs to validate kyro issues can also share this method.  In the previous `TestManifestFileSerialization` for example,  it's good to share this method.

Reply:
I agree it would be useful for other tests. We have similar test helper in Spark for roundTripKryoSerializer.

Question / nit: Does the name of the second argument of type `T` have to be `table`? This feels like it could be used with a lot more than just table objects. Maybe we should just change the name to something more generic like `item` or `toTest`?